src/Common/Common.Api/Admin/Dataset.cs

```diff
@@ -0,0 +1,39 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+
+namespace Microsoft.PowerBI.Common.Api.Encryption
```

Comment:
Namespace should match directory structure

Reply:
Moved to Encryption directory